tutorials/nservicebus-step-by-step/3-multiple-endpoints/tutorial.md

```diff
@@ -148,6 +148,8 @@ snippet: AddingRouting
 
 This establishes that commands of type `PlaceOrder` should be sent to the **Sales** endpoint.
 
+1. Ensure that the serializer between endpoints match. Since the **Sales** endpoint is using `.UseSerialization<SystemJsonSerializer>()', add in the same serializer for the **ClientUI** endpoint as well.
```

Comment:
I wish I could comment on unedited lines. In general though, I am not sure adding this information piece meal makes sense. I'd propose something more along the lines of adding a "configuration" heading [above](https://github.com/Particular/docs.particular.net/pull/6545/files#diff-eea5997fde074aacfd7e0e412710076c816a22f849f9a229c02d1cd41c06bddcR92)

> #### Configuration
> 
> Now that we have a project for our Sales endpoint, we need to add similar code to configure and start an NServiceBus endpoint:

Emphasizing "It's critical for the configuration between endpoints to match (especially message transport and serializer); otherwise, the endpoints would not be able to understand each other."

Then, in this section, I would make a smaller note to remind that the configuration in general (including the serializer) should be the same.

```suggestion
Reminder: As noted in [configuring new endpoints](link), ensure the configuration (e.g. serializer used) between endpoints match. 
```